tests/test_nps_generation.py

```diff
@@ -61,3 +61,14 @@ def test_train_model():
         "--sample_size", "200",
     ]
     train_model_main(args_list)
+
+
+def test_calculate_outcomes():
+    sampled_file = os.path.join(test_dir, "sample-1-SMILES.smi")
+
+    args_list = [
+        '--original_file', original_file,
+        '--output_dir', test_dir,
```

Comment:
We should use `temp_dir` as the output directory just like the other tests.

Reply:
Same comment for `--output_dir` in `test_train_model` as wel..

Reply:
Changed both cases to account for temp_dir.